packages/runtime/container-runtime/src/connectionTelemetry.ts

```diff
@@ -63,10 +69,40 @@ class OpPerfTelemetry {
         this.deltaManager.on("disconnect", () => {
             this.opSendTimeForLatencyStatisticsForMsnStatistics = undefined;
             this.clientSequenceNumberForLatencyStatistics = undefined;
+            this.clientSequenceNumberForSittingInProcessingQueue = undefined;
             this.connectionOpSeqNumber = undefined;
             this.firstConnection = false;
         });
 
+        this.deltaManager.outbound.on("push", (messages) => {
+            for (const msg of messages) {
+                if (msg.type === MessageType.Operation && msg.clientSequenceNumber % numOpsToThrottle === 1) {
+                        this.clientSequenceNumberForSittingInProcessingQueue = msg.clientSequenceNumber;
+                        this.opTimeSittingInInboundQueue = Date.now();
+                }
+            }
+        });
+
+        this.deltaManager.inbound.on("push", (message: ISequencedDocumentMessage) => {
+            if (message.type === MessageType.Operation) {
+                if (this.clientId === message.clientId &&
+                    this.clientSequenceNumberForSittingInProcessingQueue === message.clientSequenceNumber &&
+                    this.opTimeSittingInInboundQueue !== undefined) {
+                    const durationSittingInInboundQueue = Date.now() - this.opTimeSittingInInboundQueue;
+                    this.clientSequenceNumberForSittingInProcessingQueue = undefined;
+                    this.opTimeSittingInInboundQueue = undefined;
+                    this.logger.sendPerformanceEvent({
+                        eventName: "OpSittingInBoundQueue",
+                        sequenceNumber: message.clientSequenceNumber,
+                        referenceSequenceNumber: message.referenceSequenceNumber,
+                        duration: durationSittingInInboundQueue,
+                        category: "performance",
```

Comment:
You should remove category - it's performance by default, since sendPerformanceEvent() is used